public/components/facets/GeocoordinateFacet.vue

```diff
@@ -85,6 +85,7 @@ import {
   NUMERICAL_SUMMARY,
   RowSelection,
   SummaryMode,
+  TaskTypes,
```

Comment:
What do you need this for? If you need it, it should be in the `components/ViewTypeToggle.vue `

Reply:
removed.